src/mui/field/ChipField.js

```diff
@@ -3,23 +3,39 @@ import PropTypes from 'prop-types';
 import get from 'lodash.get';
 import pure from 'recompose/pure';
 import Chip from 'material-ui/Chip';
+import compose from 'recompose/compose';
+import translate from '../../i18n/translate';
 
-const ChipField = ({ source, record = {}, elStyle = { margin: 4 } }) => (
+
+const ChipField = ({ source, record = {}, elStyle = { margin: 4 }, translate, translationPath }) => {
+  if (translationPath) {
+    return (
+      <Chip style={elStyle}>{translate(`${translationPath}.${get(record, source)}`)}</Chip>
```

Comment:
Compared to the former version, you're displaying a chip "...visit_tag.undefined" in case we don't have tags. As done by AOR it would be better to have no chip if get(record, source) returns undefined